packages/localize/src/number/convertZeroToSpaceForGroup.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @desc Intl uses 0 as group separator for bg-BG locale.
+ * This should be a ' '
+ * @param {{type,value}[]} formattedParts
+ * @returns {{type,value}[]} corrected formatted parts
+ */
+export function convertZeroToSpaceForGroup(formattedParts) {
```

Comment:
can we rename it to forceSpaceInsteadOfZeroForGroup to keep the "normalize" function names similar 🙏